pkg/cmd/init.go

```diff
@@ -141,23 +142,28 @@ func initServerConfig(cfg *config.MicroshiftConfig) error {
 }
 
 func initKubeconfig(cfg *config.MicroshiftConfig) error {
-	if err := util.Kubeconfig(cfg.DataDir+"/resources/kubeadmin/kubeconfig", "system:admin", []string{"system:masters"}, cfg.Cluster.URL); err != nil {
+	if err := util.Kubeconfig(filepath.Join(cfg.DataDir, "/resources/kubeadmin/kubeconfig"),
+		"system:admin", []string{"system:masters"}, cfg.Cluster.URL); err != nil {
 		return err
 	}
-	if err := util.Kubeconfig(cfg.DataDir+"/resources/kube-apiserver/kubeconfig", "kube-apiserver", []string{"kube-apiserver", "system:kube-apiserver", "system:masters"}, cfg.Cluster.URL); err != nil {
+	if err := util.Kubeconfig(filepath.Join(cfg.DataDir, "/resources/kube-apiserver/kubeconfig"),
+		"kube-apiserver", []string{"kube-apiserver", "system:kube-apiserver", "system:masters"}, cfg.Cluster.URL); err != nil {
 		return err
 	}
-	if err := util.Kubeconfig(cfg.DataDir+"/resources/kube-controller-manager/kubeconfig", "system:kube-controller-manager", []string{"system:kube-controller-manager"}, cfg.Cluster.URL); err != nil {
+	if err := util.Kubeconfig(filepath.Join(cfg.DataDir, "/resources/kube-controller-manager/kubeconfig"),
+		"system:kube-controller-manager", []string{"system:kube-controller-manager"}, cfg.Cluster.URL); err != nil {
 		return err
-	}
-	if err := util.Kubeconfig(cfg.DataDir+"/resources/kube-scheduler/kubeconfig", "system:kube-scheduler", []string{"system:kube-scheduler"}, cfg.Cluster.URL); err != nil {
+	} if err := util.Kubeconfig(filepath.Join(cfg.DataDir, "/resources/kube-scheduler/kubeconfig"),
```

Comment:
A nit: this probably should be on the next line.

Reply:
```suggestion
	} 
	if err := util.Kubeconfig(filepath.Join(cfg.DataDir, "/resources/kube-scheduler/kubeconfig"),
```

Reply:
That's so weird. This file does not look like this on my local system. There's a comment on that line instead. I've added a commit to adjust the whitespace to see if that helps with the rendering.

Reply:
And of course I was on the wrong branch. This is fixes.